app.css

```diff
@@ -267,15 +272,133 @@ footer {
   align-items: center;
   text-align: center;
 }
-.contact-button:hover {
+.cont-button:hover {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:active {
+.cont-button:active {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:disabled {
+.cont-button:disabled {
   background-color: #6276a8;
   color: white;
 }
+
+@media screen and (max-width: 720px) {
+  .nav-link {
+    display: none;
+  }
+  .logo-info {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .personal-info-detail p {
+    margin: 10px 100px;
+  }
+  .work-container-wrap {
+    display: flex;
+  }
+  .work-container {
+    margin: 70px;
+    border-radius: 24px;
+  }
+  .img-work {
+    width: 544px;
+    height: 448px;
+    object-fit: cover;
+  }
+  .description-work {
+    padding: 30px;
+  }
+  .work-header {
+    font-size: 40px;
+  }
+  .work-paragraph {
+    font-size: 24px;
+  }
+  .about-container {
+    padding: 70px;
+  }
+  .about-container-wrap {
+    display: flex;
+  }
+  .about-container-wrap-header {
+    padding-right: 60px;
+  }
+  .languages ul {
+    display: flex;
+    gap: 16px;
+  }
+  .languages ul li {
+    display: flex;
+    flex-direction: column;
+  }
+  .tech-img {
+    padding: 25px;
+    object-fit: cover;
+  }
+  .about-container-wrap-skills {
+    width: 375px;
+  }
+  .personal-info-title a {
+    padding-left: 100px;
+  }
+  .personal-social-info {
+    padding-left: 100px;
+  }
+  .nav-link {
+    display: flex;
+  }
+  .nav-link-top {
+    padding: 10px;
+    font-size: 15px;
+    text-align: center;
+    color: #344563;
+  }
+  .nav-link-top a {
+    text-decoration: none;
+    color: #344563;
+  }
+  .logo-info {
+    display: none;
+  }
+  .personal-info-par {
+    font-size: 16px;
+  }
+  .contact-details {
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+  .contact-headers {
+    padding: 0 375px;
+  }
+  .contact-form {
+    display: grid;
+    row-gap: 13px;
+    justify-content: center;
+  }
+  .contact-form input {
+    justify-content: center;
+    padding: 10px 10px;
+    width: 400px;
+    align-items: center;
+  }
+  .input-place {
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+  .contact-form textarea {
+    justify-content: center;
+    padding: 10px 10px;
+    width: 400px;
+  }
+  .contact-button {
+    justify-content: center;
+  }
```

Comment:
There appears to be so much disparity between the Figma template and your implementation. Take a look at the images below. From about 720px screen width up to a certain width, a lot of your elements have issues.

- [ ] You entire website has some space to the right that is unnecessary as can be seen from the Intro section to the Contact Me section
- [ ] The Intro section of your design has a lot of spaces on the right. Kindly fix that.
- [ ] The Work cards:
    - Have a lot of spaces below the images.
    - The text on the work cards should have some spaces below them. There should be equal margin on both sides of the cards.

  **Kindly fix these issues for ALL your cards.**
- [ ] In the About me sections, the white background should cover all the contents. Kindly fix that.
- [ ] Kindly ensure that the paragraph in the Contact section occupies some more width to look like what we have on the Figma template.
- [ ] Ensure that the background of the Contact Me section covers all its contents.

  First Image | Second Image
  :---:|:---:
  ![freddy-1](https://user-images.githubusercontent.com/36542986/161248068-1b447ac0-2d30-4479-96e3-305dff642504.jpg) | ![freddy-2](https://user-images.githubusercontent.com/36542986/161248076-61640eb3-3e1a-4512-ad83-1e0fdcf4b127.jpg)

Reply:
Good job in implementing the changes.